src/framework/theme/components/icon/icon-pack.ts

```diff
@@ -0,0 +1,23 @@
+import { NbIcon } from './icon';
+
+export interface NbIcons {
+  [key: string]: NbIcon | string;
+}
+
+export enum NbIconPackType  {
+  SVG = 'svg',
+  FONT = 'font',
+}
+
+export interface NbIconPackParams {
+  packClass?: string,
+  iconPrefix?: string,
```

Comment:
Svg pack can't use this property. And I believe it should contain word `class` somewhere.